/**:
  ros__parameters:
    update_rate: 10.0
    system_emergency_heartbeat_timeout: 0.5
    use_emergency_handling: false
    check_external_emergency_heartbeat: false
    use_start_request: false
    external_emergency_stop_heartbeat_timeout: 0.0
    stop_hold_acceleration: -1.5
    emergency_acceleration: -2.4
    stopped_state_entry_duration_time: 0.1
    nominal:
      vel_lim: 25.0
      lon_acc_lim: 5.0
      lon_jerk_lim: 5.0
      lat_acc_lim: 5.0
      lat_jerk_lim: 5.0
    on_transition:
      vel_lim: 50.0
      lon_acc_lim: 1.0
      lon_jerk_lim: 0.5
      lat_acc_lim: 1.2
      lat_jerk_lim: 0.75
